### PR TITLE
Fix the duplicated turns to play for both the players

### LINKED ISSUE #16: Fix duplicated turns for player 1 & 2
In the current version of the code, the players turn is duplicated, for example: 

```
    # player 2 goes next
    print(player_turn_str.format(player_num=2))
    player2_guess = input(f"What is the capital of {country}? ")
    # player 2 goes next
    print("Player 2's turn")
    player2_guess = getpass.getpass(prompt=f"What is the capital of {country}? ")
```
This has probably been introduced while working through conflicts between [this](https://github.com/SoniaBarasa/country-capital-quiz-game/pull/15) PR and the main branch.

### PR DESCRIPTION
**Pull request by SoniaBarasa**
* Resolves #16 

This pull request will fix the duplicated turns to play for both the players. Previously, the playing turns for both players was repeated twice, making the game ambiguous.

### CODE REVIEW

File: gameofgame.py [hunk 1]
```diff
@@ -49,21 +49,21 @@ def check_guess(guess, country):
...
     # player 2 goes next
-    print("Player 2's turn")
+   
     player2_guess = getpass.getpass(prompt=f"What is the capital of {country}? ")
 
     ##Reveal the correct answer
     capital = countries_capitals[country]
     print(f"The correct answer is {capital}.")
 
     # Check the guesses and update the scores
```
Comment (ahernank) [inline gameofgame.py:65]:
```suggestion
```

File: gameofgame.py [hunk 2]
```diff
@@ -49,21 +49,21 @@ def check_guess(guess, country):
...
-    print(f"The correct answer is {capital}.")
+
 
     # Check the guesses and update the scores
```
Comment (ahernank) [inline gameofgame.py:68]:
```suggestion
```

File: gameofgame.py [hunk 3]
```diff
@@ -49,21 +49,21 @@ def check_guess(guess, country):
     # Both Players take turns to play
     #player1 goes first
     print("Player 1's turn")
```
Comment (ahernank) [inline gameofgame.py:51]:
```suggestion
    print(player_turn_str.format(player_num=1))
```

**Review by ahernank — COMMENTED** [review]
Thanks, @SoniaBarasa. Here are some suggestions.

**Review by SoniaBarasa — COMMENTED** [review]
@ahernank  @alimanfoo thank you very much for the suggestions.  I have incorporated the suggested changes into the code.

File: gameofgame.py [hunk 4]
```diff
@@ -29,73 +31,80 @@ def check_guess(guess, country):
...
 
+
 # Initialize player scores
 
 
 player1_score = 0
 player2_score = 0
 
+# Defining the congratulatory message
+congrats_message = "Congratulations Player {winner}, you are the winner!"
```
Comment (ahernank) [inline gameofgame.py:42]:
```suggestion
```

File: gameofgame.py [hunk 5]
```diff
@@ -29,73 +31,80 @@ def check_guess(guess, country):
...
     # Ask if the players want to continue playing
     play_again = input("Do you want to play again? (y/n) ")
     if play_again.lower() != "y":
         break
-    else:
-        print("The game ended in a tie. Well played both players!")  
-print("Thanks for playing!")
\ No newline at end of file
+else:
+    print("The game ended in a tie. Well played both players!")
```
Comment (ahernank) [inline gameofgame.py:108]:
```suggestion
```

**Review by ahernank — COMMENTED** [review]
Thanks @SoniaBarasa, some comments here.

**Comment by SoniaBarasa** [comment]
Hi @ahernank @alimanfoo , I will close this pull request because the same issue will be resolved on a newly created pull request so this is a repetition.

**Pull request closed without merging**